json-pointer/json-pointer.iml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like *.iml was ignored in .gitignore or the first commit was adding *.iml file and the next adding the extension to .gitignore file